addon/mixins/sortable-item.js

```diff
@@ -292,49 +297,46 @@ export default Mixin.create({
     @method freeze
   */
   freeze() {
-    let el = this.$();
+    let el = this.element;
     if (!el) { return; }
 
-    el.css({ transition: 'none' });
-    el.height(); // Force-apply styles
+    el.style.transition = 'none';
```

Comment:
I noticed when implementing the sortable component inside my app that after the initial drag and reordering of items `freeze` is invoked on every item causing a `transition: none 0s ease 0s` style to be applied. Unfortunately it doesn't look like this is cleaned up anywhere and causes the items to stop transitioning. Was this intended?

Reply:
Are you saying your issue is due to the changes in this PR or that this issue is there no matter what?

Either way it would be much better to create an issue. A comment on a closed PR isn't going to get a lot of attention

Reply:
For me it is an issue because its stops some transitions I have in place when dragging occurs. I didn't see anywhere stated in the docs that describes that situation. However I'm not sure if its an actual issue with the addon or if it was intended. That's why I posed the question here instead of creating an issue for it, I didn't want to create an issue for something thats a feature and not a bug.

Reply:
We didn’t intend to prevent people applying transitions when dragging occurs, but looking back it seems like this fell into the “unspecified behaviour” category. The history of this `freeze` method dates back to [some fixes I added when Glimmer 1 first arrived](https://github.com/heroku/ember-sortable/pull/21) nearly 4 years ago. I don’t remember might thought processes back then but it’s totally possible there’s a better way to solve this that doesn’t involve stripping *all* transitions.